### PR TITLE
GSmartControl: add version 1.1.3

### LINKED ISSUE #7497: [Request] gsmartcontrol (help needed)
## Package Request
gsmartcontrol

### Information

**Name:** [name of new package]
gsmartcontrol

**Description:** [clear and concise details of what it is]
Hard disk drive and SSD health inspection tool. GUI interface for smartmontools

**Homepage:** [a URL/link]
https://gsmartcontrol.sourceforge.io/home/

**Download link(s):** [URL(s)/link(s)]
https://github.com/ashaduri/gsmartcontrol/releases/download/v1.1.3/gsmartcontrol-1.1.3-win64.exe
https://github.com/ashaduri/gsmartcontrol/releases/download/v1.1.3/gsmartcontrol-1.1.3-win32.exe

**Some indication of popularity/repute:** [GitHub stars/software reviews etc.]

I have a fork / branch almost ready for PR https://github.com/TonyApuzzo/Extras/tree/add-gsmartcontrol 
but I have a problem with the install in that gsmartcontrol won't run from the shim.

It does work if I run it from the start menu icon.

I don't see a way to customize the shim launcher, if someone can point me to documentation or an example that tweaks the launcher I will try to get it working from the command line.

### PR DESCRIPTION
**Pull request by TonyApuzzo**
Add new package GSmartControl

Closes #7497

Relates to #XXXX

### CODE REVIEW

**Comment by TonyApuzzo** [comment]
hmm, I apologize for all the extra commits, I rebased against your master to avoid a bunch of spurious commits.  Only 1 file was changed, the new manifest.

**Review by rashil2000 — APPROVED** [review]

**Comment by issaclin32** [comment]
The installer can be extracted by **7-zip**. Therefore we don't need to run the installer.
```
{
    "version": "1.1.3",
    "description": "Hard disk drive and SSD health inspection tool",
    "homepage": "https://gsmartcontrol.sourceforge.io/home/",
    "license": "GPL-3.0-only",
    "notes": [
        "GSmartControl bundles a version of Smartmontools but it may be out of date.",
        "If Smartmontools is installed, GSmartControl will automatically use the standalone version."
    ],
    "suggest": {
        "Smartmontools": "extras/smartmontools"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/ashaduri/gsmartcontrol/releases/download/v1.1.3/gsmartcontrol-1.1.3-win64.exe#/dl.7z",
            "hash": "f33064b67b7399bea4d106862afe3a6f36edf7200a7b6b1be68a39fed4e39e6d"
        },
        "32bit": {
            "url": "https://github.com/ashaduri/gsmartcontrol/releases/download/v1.1.3/gsmartcontrol-1.1.3-win32.exe#/dl.7z",
            "hash": "29aa1b4b71677e9e01657325bd214402dcd0724cc5a65f4ff30aab487f990792"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\gsmartcontrol_uninst.exe\" -Force -Recurse | Out-Null",
    "bin": "gsmartcontrol.exe",
    "shortcut": [
        [
            "gsmartcontrol.exe",
            "GSmartControl"
        ]
    ],
    "checkver": {
        "github": "https://github.com/ashaduri/gsmartcontrol"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/ashaduri/gsmartcontrol/releases/download/v$version/gsmartcontrol-$version-win64.exe#/dl.7z"
            },
            "32bit": {
                "url": "https://github.com/ashaduri/gsmartcontrol/releases/download/v$version/gsmartcontrol-$version-win32.exe#/dl.7z"
            }
        }
    }
}

```

**Comment by issaclin32** [comment]
The config file is stored at `$Env:AppData\gsmartcontrol`. Therefore *persist* is not needed.

(just a reminder, no need to change anything regarding to this)

File: bucket/gsmartcontrol.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "version": "1.1.3",
+    "description": "Hard disk drive and SSD health inspection tool",
+    "homepage": "https://gsmartcontrol.sourceforge.io/home/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Smartmontools": "extras/smartmontools"
```
Comment (phanirithvij) [inline bucket/gsmartcontrol.json:7]: It is `main/smartmontools` not `extras/smartmontools`
```suggestion
        "Smartmontools": "main/smartmontools"
```

Comment (phanirithvij) [reply bucket/gsmartcontrol.json:7]: Also there's https://github.com/ScoopInstaller/Extras/issues/7532 which is not yet fixed.

Comment (phanirithvij) [reply bucket/gsmartcontrol.json:7]: Also there's https://github.com/ScoopInstaller/Extras/issues/7532 which is not yet fixed.

Comment (issaclin32) [reply bucket/gsmartcontrol.json]: > It is `main/smartmontools` not `extras/smartmontools`

Thanks!

**Comment by phanirithvij** [comment]
@TonyApuzzo can you make the changes requested by issaclin32 https://github.com/ScoopInstaller/Extras/pull/7501#issuecomment-990621124 i.e. append `#/dl.7z` at those 4 "url" s, and remove the installer and uninstaller parts.

```json
    "installer": {
        "args": [
            "/S",
            "/D=\"$dir\""
        ]
    },
    "uninstaller": {
        "file": "gsmartcontrol_uninst.exe",
        "args": [
            "/S"
        ]
    },
```

**Review by phanirithvij — APPROVED** [review]
Tested, works on my pc.

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**